packages/playground/src/components/select_gateway_node.vue

```diff
@@ -92,13 +119,15 @@ async function loadNextPage() {
   nodes = await loadGatewayNodes(grid!, options);
 
   if (!nodes.length && props.customDomain && props.farmData?.country) {
+    gatewayOption.value = "country";
     options.farmId = undefined;
     options.country = props.farmData.country;
     nodes = await loadGatewayNodes(grid!, options); // search in the same country
-  }
-  if (!nodes.length && props.customDomain) {
-    options.country = options.farmId = undefined;
-    nodes = await loadGatewayNodes(grid!, options); // search in the whole network
+    if (!nodes.length && props.customDomain) {
+      gatewayOption.value = "Network";
+      options.country = options.farmId = undefined;
+      nodes = await loadGatewayNodes(grid!, options); // search in the whole network
```

Comment:
in case we have more than 50 gateways, will we load the rest gateways or only the first page?

Reply:
It loads all gateways not only the first page